src/components/ErrorBoundary.tsx

```diff
@@ -1,10 +1,16 @@
 import React from 'react'
 
-import { withStyles } from '@material-ui/styles'
+import { withStyles, WithStyles } from '@material-ui/styles'
 
 import { Paragraph, Shout } from '../typography'
 import Link from './Link'
 
+type ErrorBoundaryState = {
```

Comment:
Should be an interface (for consistency with `Props`).
Same as with props, I propose state interface don't need to start with I.
```suggestion
interface ErrorBoundaryState {
```